Jakub_Murszewski_not_MA_thesis_overview.md

```diff
@@ -0,0 +1,237 @@
+![CoverImage][Cover]
+
+# Introduction
+
+
+Simple Sentiment Analysis Tool (SSAT) is a tool dedicated to performing a sentiment analysis with the use of statistical examination upon a text.
+
+## Features
+
+
+* Determinig the number and percentage of the positive, negative and neutral tokens in the text;
+* Determinig the number and percentage of the positive, negative and neutral tokens in the text, with the automatic deletion of stopwords in the text;
+* Searching for keywords in the text and analysing only the sentences containing them;
+* Automatically viewing the data, both as text and graphs;
+* Saving the extracted data to a file
+
+# Before you start
+
+
+SSAT's script is written in Python 3 programming language and as such requires Python 3 installed on the computer. Moreover, the programme requires several Python 3 modules, namely:
+* MatPlotLib
+* Re
+* Collections
+
+Those need to be installed before using SSAT.
+
+# Installation
+
+
+Simple Sentiment Analysis Tool does not need installing. The software can be opened via the console or any Python interpreter.
+
+For the programme to work correctly, several additional files need to be present. Those are:
+
+* *.txt* file with negative sentiment words
+* *.txt* file with positive sentiment words
+* *.txt* file with stopwords
+
+All of which need to be located in the same folder as the main programme file.
+
+**The code and necessary .txt files can be downloaded from:**
+
+[Here](https://github.com/MurszewskiJ/SSAT)
+
+# User Guide
+
+
+**First thing to appear after opening the program is the following menu:**
+
+![Menu][Menu]
+
+To access one of the functions of SSAT enter the number associated wit the function and press *ENTER*
+
+### Functions
+
+
+**"1. Sentiment Analysis (with stopwords)"**
+
+
+This function will analyse the whole selected text without deleting any of its contents.
+
+Upon selection of this option, the following will appear:
+
+![image_1][image_1]
+
+To proceed, enter:
+* either the path to the file which you want to analyse (if the file is not in the same folder as the programme
+* or only the name of the file (if the file is in the same folder as the programme)
+
+
+**Note that the file you wish to analyse should be in .txt format.**

```

Comment:
Not really :) I tested:
1. an .md file - it works
2. a text file without any extension - it works too

So you could update this recommendation.